ouroboros-network/test-cddl/specs/handshake-node-to-client.cddl

```diff
@@ -14,7 +14,8 @@ msgRefuse          = [2, refuseReason]
 versionTable = { * versionNumber => nodeToClientVersionData }
 
 ; from version 2 we set 15th bit to 1
-versionNumber = 1 / 32770 / 32771 / 32772 / 32773 / 32774 / 32775 / 32776 / 32777 / 32778 / 32779 / 32780
+;               9     / 10    / 11    / 12
```

Comment:
I suggest changing `from version 2 we set 15th bit to 1` to `as of version 2 (which we no longer support) we set 15th bit to 1` or some such.